src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
```

Comment:
I'm not sure if this will work, because the list on conversation is now of type Bot, not string